templates/viz/src/localMessage.js

```diff
@@ -32,20 +32,135 @@ export const message = {
   fields: {
     dimID: [
       {
-        id: 'qt_nzqx6a0xvb',
+        id: 'qt_odsbu27hzb',
```

Comment:
Can you do this change in a separate PR, it doesn't have anything to do with the codelab, it looks like.